docs/projects/zenfs.md

```diff
@@ -103,16 +112,17 @@ through *ZenFS*. Listing the files present on the device is done as follows.
           37    Jul 20 2021 18:13:52            IDENTITY
         1586    Jul 20 2021 18:14:01            MANIFEST-000004
         6178    Jul 20 2021 18:13:52            OPTIONS-000007
-
 ```
 
-### Backing-up a ZenFS Filesystem
+#### Backup files within a ZenFS file system
 
-To backup all table and metadata files on a NVMe ZNS device to a local
-filesystem, the following command can be used.
+To backup all table and metadata files within a *ZenFS* file system to a local
```

Comment:
s/backup/back up/

"backup" is a noun, referring to the process of making a copy. "back up" is a verb.